package.json

```diff
@@ -14,7 +14,7 @@
   },
   "license": "MIT",
   "dependencies": {
-    "pg": ">=2.7.0-0 <5.0.0-0"
+    "pg": ">=2.7.0-0 <7"
```

Comment:
Change this to `">=6.1.5-0 <7.0.0-0"`, lets drop support for older modules and keep the code simple